arch/risc-v/include/bl602/chip.h

```diff
@@ -1,6 +1,9 @@
 /****************************************************************************
  * arch/risc-v/include/bl602/chip.h
  *
+ * Copyright (C) 2012, 2015 Gregory Nutt. All rights reserved.
+ * Author: Gregory Nutt <gnutt@nuttx.org>
+ *
```

Comment:
Why is this being added? Apache headers do not carry copyright.
What I mentioned before about headers is that if there are files with BSD license where only Greg is the author, in that case you can change the header to Apache so that we do not grow the number of non Apache files.

Reply:
We refer to the format here at first:
https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard#cfilestructure

> convert to Apache 2.0 (same for all headers with Greg as sole author)
Does it mean adding Greg as the sole author?

Reply:
New files coming into the project should be Apache licensed (with the header as shown in the wiki page you linked). But as you see the header should not have a copyright line.
What I mentioned before was that if you are copying an existing file from NuttX which is BSD licensed, you can change the header to Apache if the author listed in the header is Gregory Nutt. Looking at the changes, I think the bl602_oneshot_lowerhalf.h and bl602_oneshot_lowerhalf.c are the only case.

Reply:
Okay, I understand now. I will check this part of the code again, thanks alot